flink-doris-connector/src/main/java/org/apache/doris/flink/sink/BackendUtil.java

```diff
@@ -100,11 +101,22 @@ public static boolean tryHttpConnection(String host) {
             LOG.debug("try to connect host {}", host);
             host = "http://" + host;
             URL url = new URL(host);
-            HttpURLConnection co = (HttpURLConnection) url.openConnection();
-            co.setConnectTimeout(60000);
-            co.connect();
-            co.disconnect();
-            return true;
+            HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+            connection.setRequestMethod("GET");
+            connection.setConnectTimeout(2000);
+            connection.setReadTimeout(2000);
```

Comment:
Is 2 seconds too little?

Reply:
It has been adjusted to 30s, which is the same as the timeout for source to read doris.